ambassador/ambassador/config/config.py

```diff
@@ -230,6 +242,7 @@ def load_all(self, resources: Iterable[ACResource]) -> None:
         rcount = 0
 
         for resource in resources:
+            self.logger.debug("Trying to parse resource: {}".format(resource))
```

Comment:
FTR I've gotten _really_ fond of `f''` strings in Python3. 🙂

Reply:
That's much cleaner, thanks :)